### PR TITLE
Frontend: Added capability to remove user(self) from Participant Team

### PR DESCRIPTION
**Pull request by deshraj**

### CODE REVIEW

**Comment by deshraj** [comment]
screencapture-localhost-8888-web-teams-1483673274160

File: frontend/src/views/web/partials/sidebar.html [hunk 1]
```diff
@@ -14,15 +14,15 @@
                 </li>
                 <li>
                     <a class="waves-effect waves-dark text-light-gray ev-md-container" ui-sref="web.teams" ui-sref-active="side-active-link">
-                        <i class="fa fa-users side-link-icon"></i> Teams</a>
+                        <i class="fa fa-users side-link-icon"></i> My Participant Teams</a>
                 </li>
                 <li>
                     <a class="waves-effect waves-dark text-light-gray ev-md-container" ui-sref="web.challenge-main" ui-sref-active="side-active-link">
-                        <i class="fa fa-fire side-link-icon"></i> Challenge Page</a>
+                        <i class="fa fa-fire side-link-icon"></i> Show all Challenges</a>
```
Comment (aka-jain) [inline frontend/src/views/web/partials/sidebar.html]: Can we use "All Challenges" instead of using "Show All Challenges"?

Comment (deshraj) [inline frontend/src/views/web/partials/sidebar.html]: Sure, will change it. :-)

File: frontend/src/views/web/partials/sidebar.html [hunk 2]
```diff
@@ -14,15 +14,15 @@
                 </li>
                 <li>
                     <a class="waves-effect waves-dark text-light-gray ev-md-container" ui-sref="web.teams" ui-sref-active="side-active-link">
-                        <i class="fa fa-users side-link-icon"></i> Teams</a>
+                        <i class="fa fa-users side-link-icon"></i> My Participant Teams</a>
```
Comment (aka-jain) [inline frontend/src/views/web/partials/sidebar.html]: I think we were thinking to delete the Team page from sidebar, isn't it?

**Comment by aka-jain** [comment]
@deshraj : I've leave some comments, have a look please.

File: frontend/src/views/web/partials/sidebar.html (hunk 2 above)
Comment (deshraj) [inline frontend/src/views/web/partials/sidebar.html]: Sorry, my bad.

Comment (aka-jain) [inline frontend/src/views/web/partials/sidebar.html]: Or may be it's a My Participation Team section where only team in which a particular user is a member can only be listed here, is it?

Comment (deshraj) [inline frontend/src/views/web/partials/sidebar.html]: So currently it is a view to choose the team for a particular challenge. We have to make it independent where user can see the list of teams or just create a team. 

When user will try to participate in a challenge, then he will be shown a similar page from where he can select one team from list of teams or create a new team and then proceed.

Comment (aka-jain) [inline frontend/src/views/web/partials/sidebar.html]: Agree, that will be great :)

**Pull request merged**